mmocr/utils/string_util.py

```diff
@@ -0,0 +1,59 @@
+class StringStrip:
+    """Removing the leading and/or the trailing characters based on the string
+    argument passed.
+
+    Args:
+        strip (bool): Whether remove characters from both left and right of
+            the string. Default: True.
+        strip_pos (str): Which position for removing, can be one of
+            ('both', 'left', 'right'), Default: 'both'.
+        strip_str (str|None): A string specifying the set of characters
+            to be removed from the left and right part of the string.
+            If None, all leading and trailing whitespaces
+            are removed from the string. Default: None.
+    """
+
+    def __init__(self, strip=True, strip_pos='both', strip_str=None, **kwargs):
+        assert isinstance(strip, bool)
+        assert strip_pos in ('both', 'left', 'right')
+        assert strip_str is None or isinstance(strip_str, str)
+
+        self.strip = strip
+        self.strip_pos = strip_pos
+        self.strip_str = strip_str
+
+    def _strip(self, in_str):
+        if self.strip_str is None:
+            in_str = in_str.strip()
+        else:
+            in_str = in_str.strip(self.strip_str)
+
+        return in_str
+
+    def _lstrip(self, in_str):
+        if self.strip_str is None:
+            in_str = in_str.lstrip()
+        else:
+            in_str = in_str.lstrip(self.strip_str)
+
+        return in_str
+
+    def _rstrip(self, in_str):
+        if self.strip_str is None:
+            in_str = in_str.rstrip()
+        else:
+            in_str = in_str.rstrip(self.strip_str)
+
+        return in_str
```

Comment:
Uncessary wrappers? 
Take `_strip()` as an example, `in_str = in_str.strip(self.strip_str)` should be fine for every possible values including `None`.